x-pack/plugins/security/public/session/session_expired.ts

```diff
@@ -32,11 +32,11 @@ const getProviderParameter = (tenant: string) => {
 export class SessionExpired {
   constructor(private logoutUrl: string, private tenant: string) {}
 
-  logout() {
+  logout(reason = 'SESSION_EXPIRED') {
```

Comment:
We are the only consumer of this module --
What do you think about making this argument required so we don't accidentally run into the same problem in the future?

```suggestion
  logout(reason: 'SESSION_EXPIRED' | 'AUTHENTICATION_ERROR') { // see LoginPage component for valid reasons
```

Reply:
Agreed but making this argument required isn't going to solve the problem that logout reasons aren't typed and we can't guarantee that we have translated messages for each logout reason. I'll refactor this properly.